### DISCUSSION TITLE
Demo Video for Getting Started

### DISCUSSION THREAD

**Discussion by techportrero** [groepl/Obsidian-Templates#18] [Q&A] [answered]
Is there any plans to create a how-to / getting-started video for forking and using your repo in obsidian and employing the zettelkasten method using your templates? It is very extensive and certainly worth tinkering with, but I want to be able to hit the ground running as I'm sure many learners would like when they learn about zettelkasten.

**Comment by groepl** [comment, marked as answer]
> Is there any plans to create a how-to / getting-started video for forking and using your repo in obsidian and employing the zettelkasten method using your templates?

I'll put it on my wish list.

**Reply by mifhud** [reply]
Has there been any progress on the demo video for its usage? I've tried reading the README guide but still feel unclear about how to apply it in real-world scenarios. Thank you, and I hope the usage video will be created soon.

**Reply by groepl** [reply]
> I've tried reading the README guide but still feel unclear about how to apply it in real-world scenarios.

Please give me some help. Where do you struggle? And what is your level of using Zettelkasten, see: https://forum.obsidian.md/t/12-principles-for-using-zettelkasten/51679/7

Maybe a simple example for the beginning? Thank you.
